nxviz/plots.py

```diff
@@ -190,6 +198,10 @@ def __init__(
             else:
                 self.group_label_color = ["black"] * len(self.nodes)
 
+        # set text properties
```

Comment:
I'd suggest validating the fontfamily, so that we can raise a loud by nonetheless informative error message for end-users.

```suggestion
        # set text properties
        valid_fonts = ['serif', 'sans-serif', 'fantasy', 'monospace']
        if fontfamily not in valid_fonts:
            raise ValueError(f'fontfamily should be one of {valid_fonts}')
```